.github/workflows/build.yml

```diff
@@ -0,0 +1,43 @@
+name: Send Chat Message
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
Not very familiar with this syntax, but would it be easy to add `main` here for newer projects?

Reply:
Yeah you're right - I copied this from a file that was using master 😓